components/cli/cmd/cellery/cellery.go

```diff
@@ -83,12 +85,28 @@ func newCliCommand(cli cli.Cli, kubeCli kubernetes.KubeCli) *cobra.Command {
 func main() {
 	celleryCli := cli.NewCelleryCli()
 	celleryKubeCli := kubernetes.NewCelleryKubeCli()
+	var ballerinaExecutor ballerina.BalExecutor
+	moduleMgr := &util.BLangManager{}
+	// Initially assume ballerina is installed locally and try to get the ballerina executable path.
+	ballerinaExecutor = ballerina.NewLocalBalExecutor()
+	ballerinaExecutablePath, err := ballerinaExecutor.ExecutablePath()
+	if err != nil {
+		util.ExitWithErrorMessage("Failed to get ballerina executable path", err)
```

Comment:
Return the error.

Reply:
This is in the main function, hence it does not return to another parent function. Therefore should exit with error message if an error occurs.